flow-report/assets/styles.css

```diff
@@ -0,0 +1,145 @@
+.App {
+  display: grid;
+  grid-template-areas: "side report";
+  grid-template-columns: min-content auto;
+  min-height: 100vh;
```

Comment:
feels like the app should be height 100vh and then have a scroll container for the individual report? 

otherwise why have grid, we could just do fixed header and sidebar :)